src/angular/styles/_typography.scss

```diff
@@ -199,16 +199,18 @@
   --sbb-ghost-button-label-color: var(--sbb-color-granite);
   --sbb-ghost-button-label-color-disabled: var(--sbb-color-iron-alpha40);
 
-  // Selection Container
-  --sbb-selection-background-color-default: var(--sbb-color-background);
-  --sbb-selection-background-color-disabled: var(--sbb-color-milk);
 
-  --sbb-selection-border-color-default: var(--sbb-color-graphite);
-  --sbb-selection-border-color-disabled: var(--sbb-color-aluminum);
+  // Checkbox & Radiobutton
+  --sbb-selection-container-background-color-default: var(--sbb-color-background);
+  --sbb-selection-container-background-color-disabled: var(--sbb-color-milk);
+
+  --sbb-selection-container-border-color-default: var(--sbb-color-graphite);
+  --sbb-selection-container-border-color-disabled: var(--sbb-color-aluminum);
+
+  --sbb-selection-container-icon-color-default: var(--sbb-color-red);
+  --sbb-selection-container-icon-color-disabled: var(--sbb-color-granite);
+  --sbb-selection-container-border-color-focused: var(--sbb-color-weak-accent);
```

Comment:
My previous suggestion did not remove enough line breaks, sorry.

Please also remove the line breaks between other `--sbb-selection-container...` variables (e.g. line 206, 209). Otherwise prettier will complain which is why the pipeline for this PR fails.